pipelines/skylab/build_indices/BuildIndices.wdl

```diff
@@ -370,18 +374,20 @@ workflow BuildIndices {
     String organism_prefix
     String genome_short_string
     String dbsnp_version
+    String? biotypes
```

Comment:
Did you mean for this to be optional? Womtools is failing https://github.com/broadinstitute/warp/runs/5095486872?check_suite_focus=true because it is trying to coerce String? to String

Reply:
I wanted to make it optional.